scripts/mutect2_wdl/mutect2-replicate-validation.wdl

```diff
@@ -0,0 +1,118 @@
+# call pairs of replicates as a tumor-normal pair, count the number of variants (i.e. false positives) 
+# and report false positive rates
+
+import "mutect2.wdl" as m2
+
+task GatherTables {
+	Array[File] tables
+
+	command <<<
+		head -n 1 ${tables[0]} > summary.txt
+		for table in ${sep=" " tables}; do
+			tail -n +2 $table >> summary.txt						
+		done
+	>>>
+
+	runtime {
+    docker: "broadinstitute/genomes-in-the-cloud:2.2.4-1469632282"
```

Comment:
Funky indentation.  Note: IntelliJ has a WDL plug-in that isn't as robust as its Java support but *does* help with formatting and some syntax highlighting.